webofneeds/won-owner-webapp/src/main/java/won/owner/web/websocket/WonWebSocketHandler.java

```diff
@@ -369,7 +372,11 @@ private void notifyPerPush(final User user, final URI atomUri, final WonMessage
                     ObjectNode rootNode = mapper.createObjectNode();
                     rootNode.put("type", "HINT");
                     rootNode.put("atomUri", userAtom.getUri().toString());
-                    rootNode.put("connectionUri", wonMessage.getRecipientURI().toString());
+                    if (wonMessage.getRecipientURI() != null) {
+                        rootNode.put("connectionUri", wonMessage.getRecipientURI().toString());
+                    } else {
+                        logger.info("received SocketHint for atom {} without recipientURI", userAtom.getUri());
```

Comment:
please on debug level, unless this is a uncommon logline that indicates a non-exception causing misbehaviour -> then please use the warn level

Reply:
ok: warn.